openwisp-utils-qa-checks

```diff
@@ -93,17 +96,32 @@ runcheckcommit() {
   fi
 }
 
+runmakemigrationcheck() {
+  echo "Running Make Migrations Check"
+  if [ ! -f "./tests/manage.py" ]; then
+    echo "File manage.py not found"
```

Comment:
please add some words that make it explicit that the test is being skipped because of this